src/components/Person/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name, age, sex, isMarried, partnerName: pName,
+  } = person;
+
+  return (
+    // eslint-disable-next-line react/jsx-filename-extension
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {`My ${sex === 'm' ? 'wife' : 'husband'} name is ${pName}`}
```

Comment:
Will be better to create separate variable before return statement with the result of married status message
And use this variable here to display some string
Such way helps to avoid adding some code logic in jsx for better understanding final html